x/authz/msgs.go

```diff
@@ -60,6 +67,21 @@ func (msg MsgGrant) ValidateBasic() error {
 	return msg.Grant.ValidateBasic()
 }
 
+// Type implements the LegacyMsg.Type method.
+func (msg MsgGrant) Type() string {
+	return sdk.MsgTypeURL(&msg)
+}
+
+// Route implements the LegacyMsg.Type method.
+func (msg MsgGrant) Route() string {
+	return sdk.MsgTypeURL(&msg)
```

Comment:
Authz has no legacy msg handler, so this is dead code. I'm actually inclined to panic here with a user friendly message saying so. wdyt?